src/main/java/org/folio/edgecommonspring/filter/EdgeSecurityFilter.java

```diff
@@ -4,12 +4,12 @@
 
 import java.io.IOException;
 import java.util.Arrays;
-import javax.servlet.FilterChain;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
```

Comment:
Looks like this module doesn't have checkstlyle, so build doesn't fail. Nevertheless, could you please use alphabetic order for imports to be consistent with existing code?

Reply:
Thank you! I have refactored the order of imports.